suite/tests/linux/fork.expect

```diff
@@ -1,5 +1,4 @@
 parent is running under DynamoRIO
-child is running under DynamoRIO
-parent is running under DynamoRIO
```

Comment:
What happened to this line?  Actually: how is this printed today?  But if this is required to pass, how does it pass without it?  Confused.

Reply:
The parent/child swap is due to buffered stdout. The missing parent is running line is sth. else. Looking into it, slightly confused now as well.

Reply:
Ok, thanks for catching this, initially I didn't see this. Looks like the original pre-patch test had an issue, somehow the printf buffer gets to the child(1). Adding a fflush(stdout) before fork for example pre-patch eliminates the second print. 

(1) not further debugged exactly what happens instruction-level.